doc/whats_new/v0.21.rst

```diff
@@ -116,6 +116,11 @@ Support for Python 3.4 and below has been officially dropped.
   :issue:`12334` by :user:`Emmanuel Arias <eamanu@eamanu.com>`,
   `Joel Nothman`_ and `Andreas Müller`_
 
+  -|Fix| A `DecprecationWarning`` is now raised when :func:`metrics.hamming_loss` 
```

Comment:
should be |API|, please refer to other API entries (e.g., XXX is deprecated in XXX and will be removed in XXX, because XXX.)